libs/langchain-google-cloud-sql-pg/src/vectorStore.ts

```diff
@@ -327,6 +329,83 @@ class PostgresVectorStore extends VectorStore {
 
     return rows;
   }
+
+  async maxMarginalRelevanceSearch(
+    query: string,
+    options?: MaxMarginalRelevanceSearchOptions<this["FilterType"]>,
+    _callbacks?: Callbacks | undefined
+  ): Promise<Document[]> {
+    const vector = await this.embeddings.embedQuery(query);
+    return await this.maxMarginalRelevantSearchByVector(
+      vector,
+      options?.k,
+      options?.fetchK,
+      options?.lambda,
+      options?.filter
```

Comment:
To simplify this code, you could just pass through the options rather than each parameter